python/tests/test-export-tagger.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+BASELINE_DIR=${HOME}/dev/work/baseline
```

Comment:
Finding the baseline dir should probably be done by either back tracking from here or with `git rev-parse --show-toplevel` rather than relaying on it being at `~/dev/work`